src/main/java/io/javalin/serversentevent/Emitter.kt

```diff
@@ -0,0 +1,37 @@
+package io.javalin.serversentevent
+
+import java.io.IOException
+import javax.servlet.AsyncContext
+import javax.servlet.ServletOutputStream
+
+class Emitter(asyncContext: AsyncContext) {
```

Comment:
You can move this variable into the constructor: 

```suggestion
class Emitter(private var asyncContext: AsyncContext) {
```

Then you can get rid of the variable + the assignment `this.asyncContext = asyncContext` in the `init` block.